TAO/tests/IDLv4/maps/test.idl

```diff
@@ -0,0 +1,5 @@
+struct DataStruct {
+    sequence<int32> seqData;
+    map<int32, int32> mapData;
+    map<int32, int32, 4> mapData;
+};
```

Comment:
This looks like a good start, but wold probably be better to have some more variety in the key and value types. Like structs, sequences, and other maps.